python/cudf_polars/cudf_polars/experimental/io.py

```diff
@@ -106,7 +106,7 @@ def from_scan(ir: Scan) -> ScanPartitionPlan:
             # _sample_pq_statistics is generic over the bit-width of the array
             # We don't care about that here, so we ignore it.
             stats = _sample_pq_statistics(ir)  # type: ignore[var-annotated]
-            file_size = sum(float(stats[column]) for column in ir.schema)
+            file_size = sum(float(stats.get(column, 0)) for column in ir.schema)
```

Comment:
Any idea on how to address this discrepancy @wence-? The `include_file_paths` column is in the schema (as expected) not in the parquet statistics dict.

Reply:
I think this is OK. Maybe add a comment to that effect.